docs/platform/howto/byoc/delete-custom-cloud.rst

```diff
@@ -0,0 +1,55 @@
+Delete your custom cloud
+========================
+
+.. topic:: Custom clouds
+
+    A :doc:`custom cloud </docs/platform/concepts/byoc>` is your own cloud infrastructure integrated with your Aiven organization. Using a custom cloud in Aiven may be the optimal solution if you have specific business needs or project requirements, such as a strict regulatory compliance. Creating and using custom clouds in your Aiven organization requires :doc:`enabling the bring your own cloud (BYOC) feature </docs/platform/howto/byoc/enable-byoc>`.
+
+This article details how to delete your custom cloud in `Aiven Console <https://console.aiven.io/>`_.
+
+About deleting custom clouds
+============================
+
+After deleting a custom cloud, the data co-hosted in this cloud will no longer be available from the Aiven platform. The deletion only impacts resources on the Aiven site (such as cloud configuration files and the bastion service) and no cloud resources on your AWS account are touched. Make sure there are no active services using your custom cloud before deleting this cloud.
+
+Prerequisites
+-------------
+
+* Administrator's role for your Aiven organization
+* At least one :doc:`custom cloud created </docs/platform/howto/byoc/create-custom-cloud>` in your Aiven organization
+* No running services that uses a custom cloud to be deleted
+* Access to `Aiven Console <https://console.aiven.io/>`_
+
+Delete your cloud
+-----------------
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ as an administrator.
+2. Select the organization you want to use from the dropdown menu in the top right corner.
+3. From the top navigation bar, select **Admin**.
+4. From the left sidebar, select **Bring your own cloud**.
+5. In the **Bring your own cloud** view, select one of the clouds available on the list.
+6. In the selected cloud's page, use the ellipsis (**...**) menu in the top right corner.
+
+.. topic:: Result
+
+    Your custom cloud has been deleted.
+
+Check it out
+------------
+
+You can preview the updated name of your cloud by taking the following steps:
```

Comment:
Is this in the right place? It doesn't seem to be particularly related to deleting a custom cloud.